INSTALL.md

```diff
@@ -368,7 +368,16 @@ cd filter_audio
 make -j$(nproc)
 sudo make install
 ```
-
+### Sqlcipher compiling, For Fedora users and possibly others.
+```bash
+git clone https://github.com/sqlcipher/sqlcipher
+cd sqlcipher
+autoreconf -if
+./configure
+make -j$(nproc)
+sudo make install
+cd ..
+````
```

Comment:
Why it's located there? I.e. does it depend on something that toxcore depends on?